### ISSUE TITLE
Installation Guide for Hedgenext-Server without Docker

### ISSUE THREAD

**Issue by DrSkyfaR** [libnewton/hedgenext-server#4]
I am a big fan of your software and would like to install it on a bare-metal server without using Docker. However, I have noticed that the current installation guide assumes the use of Docker. Therefore, I kindly request if you could provide an alternative installation guide that allows for installing the Hedgenext-Server without Docker.

There are various reasons why some of us cannot or prefer not to use Docker, whether it be due to compatibility issues, security concerns, or other technical limitations. Having an installation guide for running the Hedgenext-Server without Docker would enable us to utilize the software in our preferred environment.

I believe that such a guide would be beneficial to many users who are seeking an alternative to the Docker installation. It would also increase the flexibility of your software, making it deployable on a wider range of systems.

Thank you in advance for your attention, and I look forward to a positive response to my feature request.

Best regards,
Christopher

**Comment by libnewton** [comment]
Hi, thanks for your feedback!
Love to hear that you like this.
But tbh, this is 99% Hedgedoc with some features taken aways and some minor additions. I can recommend just checking out there excellent docs and trying to replicate it on this repo.
Maybe Hedgedoc itself is an option for you as well.
There isn't really any documentation for this project, but when starting proper documentation I'll add non-docker instructions as well. Sadly I currently lack time writing it.

**Comment by DrSkyfaR** [comment]
Hey, thanks for your quick reply, I wasn't sure how much you might have changed in the software, and if the original documentation could still apply there. 


I will try it in any case. Thanks for the tip. :) 


Maybe you can open up a project or something with points you need help with, maybe you can find some great people from the Nextcloud community that would like to support you if there is a roadmap. :)

**Comment by libnewton** [comment]
Yeah, however this is based off the legacy Hedgedoc Codebase which is very old and very ugly. Hedgedoc V2 is well in it's working. 
I think adding new features to the extremly established and well thought-of piece of software Hedgedoc is, isn't really something I want to aim for.
I also won't switch to Hedgedoc V2 with this as I dislike their new design and philosophy (hedgedoc.dev). 
So this will be mostly about maintenance and upgrading dependencies, etc.

Making it more accessible (i.e. with docs) is probably the most important thing

**Comment by MaximeHoule** [comment]
Hey @libnewton,

Just installed it the non-Docker way following the original package instructions. Seems to work, so I thought about helping you out and helping somehow with the doc. Is HedgeDoc is still the markdown editor you think is the best to connect with NextCloud? If so, how can I help?

Maxime

**Comment by libnewton** [comment]
hi @MaximeHoule glad you like it.
Yeah, I think HedgeDoc is the best way to edit notes in collab.
If you want to create some documentation, that would be highly appreciated. You can submit them via Pull Request and put them into the README.md
